### DISCUSSION TITLE
Why are all the <script>s in the wrong place?

### DISCUSSION THREAD

**Discussion by btzdnl** [dotnet/docfx#9763] [General]
Is there any particular reason why _all_ `<script>` tags in the modern template are now in an invalid location, between `</head>` and `<body>`? I see that they were originally inside `<head>`. If there’s no good reason then I suggest moving them back, to create spec-compliant HTML.

https://github.com/dotnet/docfx/blob/fa0c3c5f2a7a098cd03696a1868b92b9e8321acc/templates/modern/layout/_master.tmpl#L41-L61

**Comment by yufeih** [comment]
Thank you @fuzzykiller for raising. This wasn't intentional, would you like to contribute a PR?

**Reply by btzdnl** [reply]
Yes, gladly!
